guides/release/templates/links.md

```diff
@@ -15,7 +15,7 @@ Router.map(function() {
 ```handlebars {data-filename=app/templates/photos.hbs}
 <ul>
   {{#each this.photos as |photo|}}
-    <li><LinkTo @route="photos.edit" @model="photo">{{photo.title}}</LinkTo></li>
+    <li><LinkTo @route="photos.edit" @model={{photo}}>{{photo.title}}</LinkTo></li>
```

Comment:
We should also prefer to use `this.model` to avoid implicit `this` fallback, something like:

```suggestion
    <li><LinkTo @route="photos.edit" @model={{this.photo}}>{{this.photo.title}}</LinkTo></li>
```

What do you think?

Reply:
Great point.